### PR TITLE
MET-185: Add documentation for measurement families in the phpclient

### PR DESCRIPTION
**Pull request by tseho**
Screenshot_2020-04-07_11-38-02

Screenshot_2020-04-07_11-38-12

### CODE REVIEW

File: content/php-client/resources/target-market-settings/measurement-families.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+### Measurement family
+
+#### Get a list of measurement families
+::: php-client-availability versions=6.0 editions=CE,EE
```
Comment (SamirBoulil) [inline content/php-client/resources/target-market-settings/measurement-families.md:4]:
```suggestion
::: php-client-availability versions=5.0 editions=CE,EE
```
Where does the 6 come from ?

File: content/php-client/resources/target-market-settings/measurement-families.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+```php
+$client = new \Akeneo\Pim\ApiClient\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+/*
+ * Returns an array like this:
```
Comment (SamirBoulil) [inline content/php-client/resources/target-market-settings/measurement-families.md:10]:
```suggestion
 * Returns an array as following:
```
?

File: content/php-client/resources/target-market-settings/measurement-families.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+You can get a complete description of the returned format [here](/api-reference.html#measurement_families_get_list).
+
+::: warning
+There is no pagination on measurement families.
+:::
+
+#### Upsert a list of measurement families 
+::: php-client-availability versions=6.0 editions=CE,EE
+
+This method allows to create or update a list of measurement families.
```
Comment (SamirBoulil) [inline content/php-client/resources/target-market-settings/measurement-families.md:65]:
```suggestion
This method allows to create or partial update a list of measurement families.
```
?

File: content/php-client/resources/target-market-settings/measurement-families.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+foreach ($responses as $response) {
+    echo $response['code'];         // Measurement family code
+    echo $response['status_code'];  // 201 => created, 204 => updated, 422 => invalid
+}
+```
+
+You can get a complete description of the expected format and the returned format [here](/api-reference.html#patch_measurement_families).
+
+::: warning
+There is a limit on the maximum number of measurement families you can store. By default this limit is set to 100.
```
Comment (SamirBoulil) [inline content/php-client/resources/target-market-settings/measurement-families.md:115]: A link to where we can change this parameter in the doc ?

File: content/php-client/resources/target-market-settings/measurement-families.md (hunk 2 above)
Comment (tseho) [reply content/php-client/resources/target-market-settings/measurement-families.md:10]: I wouldn't mind but it's written exactly like this in all the other phpclient pages.

File: content/php-client/resources/target-market-settings/measurement-families.md (hunk 3 above)
Comment (tseho) [reply content/php-client/resources/target-market-settings/measurement-families.md:65]: Same here, this is the exact wording used everywhere else.
The partial update behavior on PATCH is the same for all the endpoints, as documented here: https://api.akeneo.com/documentation/update.html

File: content/php-client/resources/target-market-settings/measurement-families.md (hunk 4 above)
Comment (tseho) [reply content/php-client/resources/target-market-settings/measurement-families.md:115]: The PR is not merged yet : https://github.com/akeneo/pim-docs/pull/1441
But you are right. I'm adding the link.

Comment (tseho) [reply content/php-client/resources/target-market-settings/measurement-families.md:115]: Hum, are you sure we should link the technical documentation there ? The configuration of limits is not for serenity ...

File: content/php-client/resources/target-market-settings/measurement-families.md (hunk 1 above)
Comment (tseho) [reply content/php-client/resources/target-market-settings/measurement-families.md:4]: From the future !
Joke aside, we need to release the v6.0 of the client for including the new measurement family endpoints. Seen with @NolwennP

Comment (NolwennP) [reply content/php-client/resources/target-market-settings/measurement-families.md:4]: Yup, those new endpoints will be released in a 6.0 version of the PHP client.

File: content/php-client/resources/target-market-settings/measurement-families.md (hunk 4 above)
Comment (NolwennP) [reply content/php-client/resources/target-market-settings/measurement-families.md:115]: Nope, please do not encourage in the API doc people to change the limit. ;)

Comment (tseho) [reply content/php-client/resources/target-market-settings/measurement-families.md:115]: Great ! Thanks @NolwennP

**Pull request merged**